reporting/api/src/main/java/org/jboss/windup/reporting/model/association/LinkableModel.java

```diff
@@ -0,0 +1,27 @@
+package org.jboss.windup.reporting.model.association;
+
+import org.jboss.windup.graph.model.WindupVertexFrame;
+import org.jboss.windup.reporting.model.LinkModel;
+
+import com.tinkerpop.blueprints.Direction;
+import com.tinkerpop.frames.Adjacency;
+import com.tinkerpop.frames.modules.typedgraph.TypeValue;
+
+@TypeValue(LinkableModel.TYPE)
+public interface LinkableModel extends WindupVertexFrame
```

Comment:
In the future, try to add a little type javadoc about the general concept of what a model is for. Helps us keep it all straight :) Good job on the method api docs though those could use a little more explanation, too.  thanks.